src/v2/controllers/FormController.js

```diff
@@ -133,6 +133,11 @@ export default Controller.extend({
   handleInvokeAction(actionPath = '') {
     const idx = this.options.appState.get('idx');
 
+    // TODO: use useInteractionCodeFlow from settings instead after OKTA-405474
+    const authClient = this.options.settings.getAuthClient();
+    const transactionMeta = authClient.transactionManager.load();
+    const useInteractionCodeFlow = !!transactionMeta?.interactionHandle;
```

Comment:
Could create a help function in `client/transactionMeta.js`.

Reply:
Done